server/libs/request.js

```diff
@@ -4,6 +4,8 @@ const fs = require('fs');
 
 const isProduction = process.env.NODE_ENV === 'production';
 
+axios.defaults.headers.common['Accept-encoding'] = 'gzip';
```

Comment:
it should be case insensitive but just in case:
```suggestion
axios.defaults.headers.common['Accept-Encoding'] = 'gzip';
```